client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { createUseStyles, useTheme } from "react-jss";
+
+const useStyles = createUseStyles({
+  pointsNotification: {
+    border: "2px solid transparent",
+    backgroundColor: "#a7c539",
```

Comment:
Can you try using the theme colors here?

Reply:
Added theme colors